__snapshots__/test.js.snap

```diff
@@ -1,5 +1,66 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`Assignment Pattern 1`] = `
+Object {
+  "classes": Array [
+    Object {
+      "kind": "object",
+      "members": Array [
+        Object {
+          "default": Object {
+            "async": false,
+            "generator": false,
+            "id": null,
+            "kind": "function",
+            "parameters": Array [
+              Object {
+                "kind": "param",
+                "type": undefined,
```

Comment:
Hmm... Can you look at why this is `undefined` instead of `null`? The AST should be JSON-serializable and be deep equal before and after `JSON.parse(JSON.stringify(ast)) == ast`, but `undefined` isn't a valid JSON value

Reply:
will do